src/api/guardian.js

```diff
@@ -0,0 +1,15 @@
+const Guardian = require("guardian-js");
+const guardian_key = process.env.GUARDIAN_KEY;
```

Comment:
this doesn't need to be a variable just pass the `process.env.GUARDIAN_KEY` straight to the constructor

Reply:
what does it hurt to make it a variable !

Reply:
no variable

Reply:
just extra line of code that can be avoided

Reply:
All of the code could be done in one file, it's just better for debugging I guess

Reply:
Unused variables have the potential to confuse a new developer reading the codebase. Remove it.

Reply:
Its used